WeScan/Common/VisionRectangleDetector.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  VisionRectangleDetector.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 28/7/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import Vision
+import Foundation
+
+/// Class used to detect rectangles from an image.
+@available(iOS 11.0, *)
+struct VisionRectangleDetector {
+    
+    /// Detects rectangles from the given image on iOS 11 and above.
+    ///
+    /// - Parameters:
+    ///   - image: The image to detect rectangles on.
+    /// - Returns: The biggest detected rectangle on the image.
+    static func rectangle(forImage image: CIImage, completion: @escaping ((Quadrilateral?) -> Void)) {
+      
+        let imageRequestHandler = VNImageRequestHandler(ciImage: image, options: [:])
+
+        // Create the rectangle request, and, if found, return the biggest rectangle (else return nothing).
+        let rectangleDetectionRequest: VNDetectRectanglesRequest = {
```

Comment:
This outter scope shouldn't be needed since we only use the `VNDetectRectanglesRequest` once.

Reply:
I'm not sure what you mean here?

If it's in regards to creating the `VNDetectRectanglesRequest` as a constant, it's to make the code clearer when passing it in to the `imageRequestHandler`.